sh/tests/auto-exec-bin_test.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+FILENAME="student/auto-exec-bin.sh"
+BINFILE="~/myBins/01exec"
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+setupbin() {
+    if [ -f ${BINFILE} ]; then
+        echo "bin already exists!"
+    else
+        mkdir -p ~/myBins
+        curl -o ~/myBins/01exec https://assets.01-edu.org/devops-branch/01exec
+        chmod +x ~/myBins/01exec
+        echo "bin installed!"
+    fi
+}
+
+challenge() {
+    # run soultion script
+    export PATH=""
```

Comment:
when i run the test i get this error:
"./auto-exec-bin_test.sh: line 28: 01exec: command not found"
The new path is never added as you can see in the print, that is my current path printed after the student script run, and it doesn't add the "~/myBins" to the path. so the test is stuck there. 
![image](https://user-images.githubusercontent.com/53977751/209825105-59046867-eb49-4f1e-b23d-bfcdf1dfc52f.png)

Reply:
can you run the test in your computer without this error?

Reply:
I replaced the 01exec with a simple bash script - the one used before was not running on my machine (Ubuntu 22.04)

Reply:
Nice!